dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -10,12 +10,20 @@ RUN apt-get update && \
     apt-get install -y g++ python-dev libxml2
 
 RUN pip install connexion==1.1.15 \
```

Comment:
Suggest replacing `pip` with `pip2` to ensure the Python 2 version is used here.